src/Makefile.am

```diff
@@ -878,6 +878,7 @@ libbitcoinkernel_la_SOURCES = \
   key.cpp \
   logging.cpp \
   node/blockstorage.cpp \
+  node/chain.cpp \
```

Comment:
88cad3cb604058d8f26790eeb37abe2dabc0c033: Could explain in the commit message that this is only temporary? If not, move it to the kernel folder?